modules/ROOT/pages/add-api-to-unified-schema.adoc

```diff
@@ -119,9 +119,30 @@ To edit the authentication method:
 . Select an authentication policy and complete any required fields.
 . Click *Save*.
 
+[[upate-an-api-version]]
```

Comment:
up**d**ate* (sorry for the little changes 😬 )

Reply:
@tfoxy don't apologize, thank you for catching that!